OUXT-Polaris/robotx_packages
# Devel/feature/ball launcher

## Status
** DEVELOPMENT**

## Description
add ball launcher xacro

## Todos


## Steps to Test or Reproduce

test with gazebo
```
roslaunch robotx_gazebo robotx_gazebo.launch
```

check URDF
```
roslaunch wamv_description description_robotx.launch
```

see also
[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/5b83t_WskGI/0.jpg)](http://www.youtube.com/watch?v=Y5b83t_WskGI)
